### PR TITLE
Fix colors to fit current CSS Color Module Level 3

### PR DESCRIPTION
**Pull request by toptalo**
While I was checking issue CSSLint/csslint#717, I've noticed some mistakes in this module and fix it to fit [CSS Color Module Level 3](https://www.w3.org/TR/css3-color/#svg-color) [Recommendation]

### CODE REVIEW

File: src/css/Colors.js [hunk 1]
```diff
@@ -164,7 +164,6 @@ var Colors = module.exports = {
     buttontext              : "Text on push buttons.",
     captiontext             : "Text in caption, size box, and scrollbar arrow box.",
     graytext                : "Grayed (disabled) text. This color is set to #000 if the current display driver does not support a solid gray color.",
-    greytext                : "Greyed (disabled) text. This color is set to #000 if the current display driver does not support a solid grey color.",
```
Comment (stubbornella) [inline src/css/Colors.js]: Why remove this?

Comment (toptalo) [reply src/css/Colors.js]: I've just seen difference. Thanks.

**Comment by frvge** [comment]
lgtm. Thanks for your contribution @toptalo .

**Comment by toptalo** [comment]
Thanks!

**Pull request merged**